config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -27,14 +27,21 @@ periodics:
           memory: "2Gi"
 
 - name: ci-test-infra-triage
-  interval: 1h
+  decorate: true
+  interval: 20m
   spec:
     containers:
     - image: gcr.io/k8s-testimages/triage:latest
       imagePullPolicy: Always
       env:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
+      command:
+        - "timeout"
+      args:
+        - "-t"
```

Comment:
what is -t? I don't see this in the manpages

Reply:
That's apparently because this is busybox for some reason
```
$ docker run -ti gcr.io/k8s-testimages/triage timeout -h
timeout: unrecognized option: h
BusyBox v1.27.2 (2018-06-06 09:08:44 UTC) multi-call binary.

Usage: timeout [-t SECS] [-s SIG] PROG ARGS

Runs PROG. Sends SIG to it if it is not gone in SECS seconds.
Defaults: SECS: 10, SIG: TERM.
```

Reply:
ahhh, right. alpine